devices/siglis.js

```diff
@@ -0,0 +1,42 @@
+const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
+const tz = require('zigbee-herdsman-converters/converters/toZigbee');
+const exposes = require('zigbee-herdsman-converters/lib/exposes');
+const reporting = require('zigbee-herdsman-converters/lib/reporting');
+const e = exposes.presets;
+
+
+module.exports = [
+    {
+        zigbeeModel: ['zigfred uno'],
+        model: 'zigfred uno',
+        vendor: 'Siglis',
+        description: 'Smart In-Wall Switch',
+        exposes: [e.light_brightness_colorxy().withEndpoint('l1'), e.switch().withEndpoint('l2'), e.light_brightness().withEndpoint('l3')],
+        fromZigbee: [fz.color_colortemp, fz.on_off, fz.brightness, fz.level_config, fz.power_on_behavior, fz.ignore_basic_report],
```

Comment:
Please use `extend:` instead of `fromZigbee`/`toZigbee`

Reply:
Thanks for your review! I am struggling to see how I could use `extend: ` instead of `fromZigbee/toZigbee` cleanly, as this is a multi-endpoint composite device which has to expose three different device types `light_brightness_colorxy`, `switch`, `light_brightness` on three different endpoints.
Also, I can not find any similar examples in existing converters. Do you have a suggestion how to handle this situation?

Reply:
On second thought it might work this way, because `light_onoff_brightness_color` is a superset of the neccesary zigbee messages and `exposes ` will be overwritten. Do you agree?
```js
extend: extend.light_onoff_brightness_color(),
exposes: [e.light_brightness_colorxy().withEndpoint('l1'), e.switch().withEndpoint('l2'), e.light_brightness().withEndpoint('l3')],
```
I have no access to my test setup until monday, unfortunately.

Reply:
> as this is a multi-endpoint composite device which has to expose three different device types

Missed that, sorry!